src/pages/starter-kit/checkout/tax-use-cases.md

```diff
@@ -115,6 +115,28 @@ The Appbuilder application receives the following payload as an `oopQuote` objec
 }
 ```
 
+### Tax inclusive vs tax exclusive pricing
+
+Adobe Commerce supports two pricing models for tax calculation: tax-inclusive and tax-exclusive pricing. The `is_tax_included` flag in the webhook payload indicates the model used for each line item:
+
+- `is_tax_included: true` - The line item's `unit_price` already includes the full tax amount, which means that the [App Builder webhook](https://github.com/adobe/commerce-checkout-starter-kit/blob/main/actions/collect-taxes/index.js#L84) must avoid re‑adding the same tax.
```

Comment:
I think this part is not correct - "the [App Builder webhook] must avoid re‑adding the same tax" .
App Builder should set the tax amount correctly, not zero. oope-tax modules doesn't sum it up to the subtotal instead. 
THis is what our partner is doing not as expected, as they were setting 0 for VAT, which shouldn't.
https://jira.corp.adobe.com/browse/CEXT-5363